elfeed-org.el

```diff
@@ -36,6 +36,8 @@
 (require 'org)
 (require 'dash)
 (require 's)
+(require 'cl-lib)
+(require 'xml)
```

Comment:
Searching melpa packages under my .emacs.d just now, looks only `cl-lib` should be append to `Package-Requires`, I'd like to use `(cl-lib "0.5")` like most did, but my is `1.0`. I will check the difference later.